### PR TITLE
fix: Fix changed namespacegroup_id, namespace_id param check

### PR DESCRIPTION
**Pull request by kang2453**
### Category
- [ ] New feature
- [x] Bug fix
- [ ] Improvement
- [ ] Refactor
- [ ] etc

### Description
   fix: Fix changed namespacegroup_id, namespace_id param check
### Known issue

### CODE REVIEW

File: src/spaceone/inventory_v2/manager/namespace_group_manager.py [hunk 1]
```diff
@@ -30,8 +30,10 @@ def _rollback(vo: NamespaceGroup):
             )
             vo.delete()
 
-        if "namespace_group_id" not in params:
+        namespace_group_id = params.get("namespace_group_id")
+        if namespace_group_id is None:
```
Comment (ImMin5) [inline src/spaceone/inventory_v2/manager/namespace_group_manager.py]:
```suggestion
        if namespace_group_id := params.get("namespace_group_id"):
```

Comment (ImMin5) [reply src/spaceone/inventory_v2/manager/namespace_group_manager.py]: This is not a mandatory change. It's a code example aimed at reducing the number of lines.

File: src/spaceone/inventory_v2/model/namespace_group/request.py [hunk 2]
```diff
@@ -39,14 +39,14 @@ class NamespaceGroupUpdateRequest(BaseModel):
 
 class NamespaceGroupDeleteRequest(BaseModel):
     namespace_group_id: str
-    resource_group: ResourceGroup
+    resource_group: Union[ResourceGroup,None] = None
```
Comment (ImMin5) [inline src/spaceone/inventory_v2/model/namespace_group/request.py]:
```suggestion
```

File: src/spaceone/inventory_v2/model/namespace_group/request.py [hunk 3]
```diff
@@ -39,14 +39,14 @@ class NamespaceGroupUpdateRequest(BaseModel):
...
     workspace_id: Union[str,None] = None
     domain_id:str
 
 
 class NamespaceGroupGetRequest(BaseModel):
     namespace_group_id: str
-    resource_group: ResourceGroup
+    resource_group: Union[ResourceGroup,None] = None
```
Comment (ImMin5) [inline src/spaceone/inventory_v2/model/namespace_group/request.py]:
```suggestion
```

File: src/spaceone/inventory_v2/manager/namespace_group_manager.py (hunk 1 above)
Comment (kang2453) [reply src/spaceone/inventory_v2/manager/namespace_group_manager.py]: ok changed
```suggestion
        if namespace_group_id := params.get("namespace_group_id"): 
            params["namespace_group_id"] = utils.generate_id("nsg")
```

File: src/spaceone/inventory_v2/model/namespace_group/request.py (hunk 3 above)
Comment (kang2453) [reply src/spaceone/inventory_v2/model/namespace_group/request.py]: ok. remove resource_group

File: src/spaceone/inventory_v2/model/namespace_group/request.py (hunk 2 above)
Comment (kang2453) [reply src/spaceone/inventory_v2/model/namespace_group/request.py]: ok. remove resource_group

**Pull request merged**